docs/getting-started.md

```diff
@@ -33,3 +73,29 @@ There are plenty of workflow examples using Cumulus tasks [here](https://github.
 There is a list of Cumulus tasks already included in your deployment [here](tasks).
 
 After configuring your workflow and redeploying, you can configure and run your workflow using the same steps as in step 2.
+
+---
+
+## Helpful Tips
+
+Here are some useful tips to keep in mind when deploying or working in Cumulus.
+
+### Integrator/Developer
+
+* [Versioning and Releases](https://github.com/nasa/cumulus/blob/master/docs/development/release.md): This documentation gives information on our global versioning approach. We suggest upgrading to the supported version for Cumulus, Cumulus dashboard, and Thin Egress App (TEA).
```

Comment:
```suggestion
* [Versioning and Releases](deployment/upgrade-readme): This documentation gives information on our global versioning approach. We suggest upgrading to the supported version for Cumulus, Cumulus dashboard, and Thin Egress App (TEA).
```
The `release.md` doc is internal to the team since most people would not have the permissions to actually do a release.